src/Data/JSON/QQ.hs

```diff
@@ -115,19 +116,19 @@ symbol :: CharParser () String
 symbol = many1 (noneOf "\\ \":;><${}")
 
 commaSep :: CharParser () a -> CharParser () [a]
-commaSep p  = p `sepBy` (char ',')
+commaSep p  = p `sepBy` char ','
 
 chars :: CharParser () String
 chars = do
-       try (string "\\\"" *> pure "\"")
-   <|> try (string "\\\\" *> pure "\\")
-   <|> try (string "\\/" *> pure "/")
-   <|> try (string "\\b" *> pure "\b")
-   <|> try (string "\\f" *> pure "\f")
-   <|> try (string "\\n" *> pure "\n")
-   <|> try (string "\\r" *> pure "\r")
-   <|> try (string "\\t" *> pure "\t")
-   <|> try (unicodeChars)
+       try (string "\\\"" $> "\"")
+   <|> try (string "\\\\" $> "\\")
+   <|> try (string "\\/" $> "/")
+   <|> try (string "\\b" $> "\b")
+   <|> try (string "\\f" $> "\f")
+   <|> try (string "\\n" $> "\n")
+   <|> try (string "\\r" $> "\r")
```

Comment:
👍 but please also adjust the `base` lower bound accordingly

Reply:
> 4.7.0.0 Apr 2014
> ...
> Data.Functor now exports ($>) and void.
> SOURCE: [base changelog](https://hackage.haskell.org/package/base-4.18.0.0/changelog)

Reply:
I bumped base to match.